### DISCUSSION TITLE
How to set default scalacOptions on the mill command line?

### DISCUSSION THREAD

**Discussion by olhotak** [com-lihaoyi/mill#1603] [Q&A]
I'm trying to tweak the options passed to scalac in the Scala 3 Community Build, which consists of both sbt projects and mill projects. Each project builds with its own sbt or mill build file that comes unmodified from the upstream project. The Community Build script exec's a new sbt or mill instance for each project. For sbt projects, the script can specify default options for scalac using `set Global/scalacOptions` on the sbt command line. Is there a way to do the same for mill? I'm trying to avoid having to patch the mill build files of the individual projects.

**Comment by lefou** [comment]
This is not supported by Mill.
